.github/ISSUE_TEMPLATE/tracking-issue.md

```diff
@@ -0,0 +1,24 @@
+___
+name: Tracking Issue
+about: Create a meta issue for tracking more granular tasks
+___
+
+## Background
+
+<!-- Short description of the motivation and relevant background for this effort -->
+
+## Tasks
+
+<!-- Add tasks/issues that contribute to this effort here -->
+
+- [ ] 
+
+____
+
+#### For Admin Use
+
+- [ ] Not duplicate issue
+- [ ] Appropriate labels applied
```

Comment:
We should refine this expectation here. The "appropriate label" is probably a specific one in the case of tracking issues, right? Probably https://github.com/informalsystems/hermes/labels/admin or https://github.com/informalsystems/hermes/labels/okr ?

Reply:
I added a note at the top of the template. Hopefully that makes it clear enough which label is appropriate for which situation.